db/bigtable_eth1.go

```diff
@@ -1064,10 +1064,6 @@ func (bigtable *Bigtable) TransformTx(blk *types.Eth1Block, cache *freecache.Cac
 			isContract = true
 		}
 		// logger.Infof("sending to: %x", to)
-		invokesContract := false
-		if len(tx.GetItx()) > 0 || tx.GetGasUsed() > 21000 || tx.GetErrorMsg() != "" {
-			invokesContract = true
-		}
```

Comment:
This code is also used for blob tx:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/929fbe89b0166a2d4a255cccd83593e115617961/db/bigtable_eth1.go#L1162-L1165
Should it be updated there as well?

Reply:
Yes, good catch :+1: